plane/src/types/backend_state.rs

```diff
@@ -68,6 +86,97 @@ pub enum BackendState {
     },
 }
 
+impl valuable::Valuable for BackendState {
```

Comment:
this creates log entries of the form `{"status": "scheduled"}` or `{"status": "waiting", "address": "..."}`